rate/static/README.txt

```diff
@@ -0,0 +1,19 @@
+This static directory is for files that should be included in your kit as plain
+static files.
+
+You can ask the runtime for a URL that will retrieve these files with:
+
+    url = self.runtime.local_resource_url(self, "static/js/lib.js")
+
+The default implementation is very strict though, and will not serve files from
```

Comment:
@andy-armstrong when you are crawling xblocks for resources, does it matter what directory they are in (are you expecting them to be in a "public" directory)?